oap-server/server-bootstrap/src/main/resources/application.yml

```diff
@@ -366,7 +366,7 @@ kafka-fetcher:
     namespace: ${SW_NAMESPACE:""}
     partitions: ${SW_KAFKA_FETCHER_PARTITIONS:3}
     replicationFactor: ${SW_KAFKA_FETCHER_PARTITIONS_FACTOR:2}
-    enableMeterSystem: ${SW_KAFKA_FETCHER_ENABLE_METER_SYSTEM:false}
+    enableMeterSystem: ${SW_KAFKA_FETCHER_ENABLE_METER_SYSTEM:true}
```

Comment:
As we discussed, I think we should remove this permanently. The whole config. Because according to your description, there is always a bug if this is set to false.

Reply:
> As we discussed, I think we should remove this permanently. The whole config. Because according to your description, there is always a bug if this is set to false.

@wu-sheng it will always check the meter topic at agent side. what you say is more reasonable. I will remove the <code> enableMeterSystem</code> switch at oap side.